inst/tests/tests.Rraw

```diff
@@ -15870,6 +15870,10 @@ test(2097.1, DT[.(NA_character_)], data.table(fac=NA_character_, v=2L, key="fac"
 test(2097.2, DT[J(NA)],          error="Factor columns must join to factor or character columns")
 test(2097.3, DT[J(NA_integer_)], error="Factor columns must join to factor or character columns")
 
+# expression columns in rbindlist #546
+a <- data.table(c1 = 1, c2 = 'asd', c3 = expression(as.character(Sys.time())))
+b <- data.table(c1 = 3, c2 = 'qwe', c3 = expression(as.character(Sys.time()+5)))
+test(2098, rbind(a,b)$c3, expression(as.character(Sys.time()), as.character(Sys.time()+5)))
```

Comment:
I don't find this useful at all. The new expression doesn't even evaluate. I am against combining language objects in rbindlist. The useful scenario I had in the past was to have them in a list column, which at that time I was not aware.